style-guide/advice.md

```diff
@@ -0,0 +1,33 @@
+# Other style advice
+
+## Expressions
+
+Prefer to use Rust's expression oriented nature where possible;
+
+```rust
+// use
+let x = if y { 1 } else { 0 };
+// not
+let x;
+if y {
+    x = 1;
+} else {
+    x = 0;
+}
+```
+
+## Names
+
+ * Types shall be `PascalCase`,
```

Comment:
[It was decided](https://github.com/rust-lang/rfcs/pull/2389#issuecomment-384313272) to use `UpperCamelCase` rather than `PascalCase`.